index.js

```diff
@@ -59,7 +65,13 @@ module.exports = (input, opts) => {
 				task: () => del('node_modules')
 			},
 			{
-				title: 'Installing dependencies',
+				title: 'Installing dependencies using Yarn',
+				enabled: () => opts.yarn === true,
+				task: () => exec('yarn', ['install'])
```

Comment:
This can happen when a dependency gets upgraded (using NPM perhaps) but the lockfile wasn't regenerated. 

I'm not quite sure what the best course of action would be, maybe failing isn't so bad in hindsight 🤔